plugins/att_cybersecurity_alienvault_otx/plugin.spec.yaml

```diff
@@ -3,9 +3,9 @@ extension: plugin
 products: [insightconnect]
 name: att_cybersecurity_alienvault_otx
 title: AT&T Cybersecurity AlienVault Open Threat Exchange
-description: This plugin utilizes AT&T Cybersecurity (AlienVault OTX) to retrieve details about an indicator
- Open Threat Exchange is the neighborhood watch of the global intelligence community
-version: 2.0.0
+description: This plugin utilizes AT&T Cybersecurity (AlienVault OTX) to retrieve details about an indicator. Open Threat Exchange is the neighborhood watch of the global intelligence community
```

Comment:
```suggestion
description: This plugin utilizes AT&T Cybersecurity (AlienVault OTX) to retrieve details about an indicator. Open Threat Exchange is the neighborhood watch of the global intelligence community.
```

Reply:
We shouldn't add a period at the end of the description.